avaframe/com1DFA/com1DFA.py

```diff
@@ -1447,9 +1456,76 @@ def DFAIterate(cfg, particles, fields, dem, simHash=''):
                                                         fields, mtiInfo)
         dtAna.exportData(mtiInfo, cfgRangeTime, 'com1DFA')
 
+    # save resDF to file
+    resDFPath = pathlib.Path(cfgGen['avalancheDir'], 'Outputs', 'com1DFA', 'resDF_%s.csv' % simHash)
+    resDF.to_csv(resDFPath)
+
     return Tsave, particlesList, fieldsList, infoDict
 
 
+def setupResDF(resTypes, cfgRangeTime):
+    """ setup result fields max values dataframe for initial time step
+        for all resTypes used and optional for avalanche front
+
+        Parameters
+        -----------
+        resTypes: list
+            list of all resultTypes
+        cfgRangeTime: bool
+            config info if range time diagram should be performed and rangeList is available
+
+        Returns
+        --------
+        resDF: dataframe
+            data frame with on line for iniital time step and max and mean values of fields
+    """
+
+    resDict = {'timeStep': [0.0]}
+    for resT in resTypes:
+        if resT != 'particles':
+            resDict['max' + resT] = [0.0]
+    if cfgRangeTime:
+        resDict['rangeList'] = [0.0]
+    resDF = pd.DataFrame.from_dict(resDict)
+    resDF = resDF.set_index('timeStep')
+
+    return resDF
+
+
+def getMaxMeanValues(resDF, fields, timeStep, resTypes, rangeValue=''):
```

Comment:
We already have a getMaxMeanValues function in aimecTools, could that be used?

Reply:
Actually the getMaxMeanValues function here just appends the max value of the fields to the dataFrame and if provided the rangeValue, the actual computation of the max value is just one line and so I think not useful to call the aimec function, however, the function here could be renamed to addMaxValuesToDF?